frontend/src/metabase/admin/datamodel/containers/MetricForm.jsx

```diff
@@ -51,7 +51,7 @@ import Table from "metabase-lib/lib/metadata/Table";
       return errors;
     },
   },
-  (state, { metric }) => ({ initialValues: metric }),
+  (state, { metric }) => ({ initialValues: metric && metric._plainObject }),
```

Comment:
I think I would prefer if we assumed `MetricForm` took the non-wrapped object and instead unwrapped it in `MetricApp`:

```js
@Metrics.load({
  id: (state, props) => parseInt(props.params.id),
  wrapped: true,
})
@Tables.load({ id: (state, props) => props.metric.table_id, wrapped: true })
@withTableMetadataLoaded
class UpdateMetricForm extends Component {
  onSubmit = async metric => {
    await this.props.metric.update(metric);
    MetabaseAnalytics.trackEvent("Data Model", "Metric Updated");
    const { id: tableId, db_id: databaseId } = this.props.table;
    this.props.onChangeLocation(
      `/admin/datamodel/database/${databaseId}/table/${tableId}`,
    );
  };

  render() {
    const { metric, ...props } = this.props;
    return <MetricForm {...props} metric={metric.getPlainObject()} onSubmit={this.onSubmit} />;
  }
}
```

Reply:
👍I like that more.

Unrelated to the fix, but I also unwrapped them (in the entity sense) since those methods were only used once.